### PR TITLE
Add CDK Test Project to test Macro 

### PR DESCRIPTION
**Pull request by ssamueluk**
Added CDK Test project.

## Description
Added CDK Test project.

## Motivation and Context
To test SlicWatch-Macro in CDK stacks

## How Has This Been Tested?
Deployed cdk stacks and checked whether alarms and dashboards are automatically getting created.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [x ] My change requires a change to the documentation.
- [ x] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING.md** document.
- [ ] I have added tests to cover my changes.
- [ x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +132,23 @@ See the [Configuration](#configuration) section below for more detailed instruct
 
 If you want to override the default alarm and dashboard settings for each Lambda Functino resource, add the `slicWatch` property to the `Metadata` section.
 
+### Adding the SLIC Watch Transform to CDK
+Once you have deployed the macro, add it to CDK Stack 
+
+```
+this.addTransform("SlicWatch-v2")
```
[the remark this answers is not in the document]
Comment (ssamueluk) [reply README.md:139]: updated.

File: README.md [hunk 2]
```diff
@@ -130,6 +132,23 @@ See the [Configuration](#configuration) section below for more detailed instruct
...
+```
+🪛 _Optionally_, add some configuration for the plugin as below:
+
+```
+this.templateOptions.metadata = {
+      "slicWatch": {
```
[the remark this answers is not in the document]
Comment (ssamueluk) [reply README.md]: updated

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ssamueluk) [reply README.md:141]: It is bit going to take time as there is no proper documentation that we can use , have to go through the language specific SDK's. I've now tested adding transforms in different languages and updated the documentation, but not sure I should spent time in testing and adding detailed configuration instructions?

File: cdk-test-project/bin/cdk-test-project.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
```
[the remark this answers is not in the document]
Comment (ssamueluk) [reply cdk-test-project/bin/cdk-test-project.ts:2]: I've added eslint typescript plugin to cdk test project now. Is this what you mean?

File: cdk-test-project/resp.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'index'\nRequire stack:\n- /var/runtime/index.mjs","trace":["Runtime.ImportModuleError: Error: Cannot find module 'index'","Require stack:","- /var/runtime/index.mjs","    at _loadUserApp (file:///var/runtime/index.mjs:951:17)","    at async Object.UserFunction.js.module.exports.load (file:///var/runtime/index.mjs:976:21)","    at async start (file:///var/runtime/index.mjs:1137:23)","    at async file:///var/runtime/index.mjs:1143:1"]}
```
Comment (eoinsha) [inline cdk-test-project/resp.json:1]: @ssamueluk I assume we can remove this file now

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eoinsha) [reply README.md]: No problem - we can leave it

**Review by eoinsha — CHANGES_REQUESTED** [review]

File: cdk-test-project/bin/cdk-test-project.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (eoinsha) [reply cdk-test-project/bin/cdk-test-project.ts]: Yes, that's great. I pushed a change to the TS eslint configuration to align with the [standardjs](https://standardjs.com/) used in the rest of the project.

File: cdk-test-project/resp.json (hunk 4 above)
Comment (ssamueluk) [reply cdk-test-project/resp.json]: Sure, removed the file.

**Pull request merged**